core/src/main/java/org/bitcoinj/wallet/Wallet.java

```diff
@@ -2383,13 +2383,10 @@ private void receive(Transaction tx, StoredBlock block, BlockChain.NewBlockType
         }
 
         informConfidenceListenersIfNotReorganizing();
-        if(CreditFundingTransaction.isCreditFundingTransaction(tx)) {
-            CreditFundingTransaction cftx = getCreditFundingTransaction(tx);
-            queueOnCreditFundingEvent(cftx, block, blockType);
-        }
+
         // let keychain extensions process the transaction
         for (KeyChainGroupExtension extension : keyChainExtensions.values()) {
-            extension.processTransaction(tx);
+            extension.processTransaction(tx, block, blockType);
         }
```

Comment:
handing credit funding transactions is now part of the AuthenticationGroupExtension.